tests/analyses/test_api.py

```diff
@@ -382,14 +382,11 @@ async def test_remove(
             }
         )
 
-    m_remove = mocker.patch("virtool_core.utils.rm")
-
     resp = await client.delete("/analyses/foobar")
 
     match error:
         case None:
             await resp_is.no_content(resp)
-            assert m_remove.called_with("data/samples/baz/analyses/foobar", True)
```

Comment:
This is eliminating part of the test. Am I missing something?

Reply:
the mock was never being called as checked with m_remove.call_count being 0 and m_remove.call_args (the way to check arguments passed to magic mocks) being None. Since the mock was never used, I removed the code to clean it up.